src/replay-server.ts

```diff
@@ -23,13 +23,14 @@ let recordingLogged = false;
  * "replayRoot/testName/queryName-queryVarsHash". Otherwise, the responses are
  * read back from those files.
  */
-export async function startTestReplayServer(t: ava.ExecutionContext):
-    Promise<{server: http.Server, url: string}> {
+export async function startTestReplayServer(
+    t: ava.ExecutionContext,
+    title?: string): Promise<{server: http.Server, url: string}> {
   const record = process.env.RECORD === 'true';
 
   // Kind of weirdly, t.title will include the name of the current function
   // plus " for " as a prefix. Get the original name instead.
-  const testTitle = t.title.replace(/^[\w\s]+ hook for /, '');
+  const testTitle = title ? title : t.title.replace(/^[\w\s]+ hook for /, '');
```

Comment:
This line is now a no-op, since you only run the replace when there is nothing to replace.

Reply:
Oh, nevermind, misread. Now I'm just a little confused about what why there's both title and t.title.